glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -333,12 +346,13 @@ namespace
 
                 for (auto &primitive : mesh.primitives)
                 {
-                    AddIndexOffset(primitive.positionsAccessorId, accessorOffset);
-                    AddIndexOffset(primitive.normalsAccessorId, accessorOffset);
                     AddIndexOffset(primitive.indicesAccessorId, accessorOffset);

```

Comment:
Is this correct? or does it have to be changed as well?

Reply:
Indices is still its own field where empty string implies not present in the json. So no it doesn't need to change.